### ISSUE TITLE
Aztec: Data Loss

### ISSUE THREAD

**Issue by jleandroperez** [wordpress-mobile/WordPress-iOS#8399]
### Description:
We've received a data loss report from one of our users. Repro steps below!

### Details:
1. Launch WPiOS
2. Open Aztec and paste the sample HTML snippet attached below
3. Publish the post
4. Verify that the post shows up correctly
5. Edit the post in Aztec: Perform no changes, but hit the **update** button
6. Refresh the post in your browser

As a result, you'll notice the initial characters in the list go missing.

cc @diegoreymendez 

--

```
Test &amp; Test 😊<strong>Test &amp; Test</strong>&lt;


1 tablespoon olive oil&lt;	 	 
 lb. sweet Italian sausage, removed from casings&lt;	 	 
 medium carrots, peeled, sliced into thin half-moons&lt;	 	 
 stalks celery, trimmed, thinly sliced&lt;	 	 
 medium yellow onion, diced small&lt;	 	 
 cloves garlic, minced&lt;	 	 
/4 cup tomato paste&lt;	 	 
 bay leaves&lt;	 	 
 tablespoon fresh thyme leaves (or 1/2 teaspoon dried)&lt;	 	 
/4 teaspoon red pepper flakes, optional&lt;	 	 
 cups chicken stock&lt;	 	 
-4 cups water (as needed)&lt;	 	 
 15-ounce cans cannellini beans, drained and rinsed (about 3 cups)&lt;	 	 
 bunch lacinato kale, cleaned, stems removed and torn into pieces&lt;	 	 
/4 cup chopped fresh parsley&lt;	 	 
osher or sea salt to taste, if needed
```

**Comment by SiobhyB** [comment]
In addition to 250592-h, this has come up again in 251202-h. (Noting for my own reference, so I can follow up with the users when this is fixed.)

**Comment by SiobhyB** [comment]
Again in 251637-h.

**Comment by jleandroperez** [comment]
hello there @SiobhyB!,

PR merged, this will be fixed in WPiOS Mark 9.2. Thanks for all the help!!

**Comment by SiobhyB** [comment]
@jleandroperez: Is there any way the fix could be rolled out sooner? We're still getting reports of this issue and I'm finding it hard to: 1. Let people know that they've lost content due to a bug we know exists and have had a fix for for 14 days. 2. Ask them to wait until the 30th January.
